Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -163,6 +163,9 @@ typedef NS_ENUM(NSInteger, _WKImmediateActionType) {
 
 @interface WKWebView (WKPrivate)
 
+@property (nonatomic, readonly) NSString *_proxyName;
+@property (nonatomic, readonly) BOOL _isPageLoadCommitted;
```

Comment:
I don't think we'll need this.

Reply:
This let's the client check if the call back has already been called and in case they want to use the other values somewhere other than the callback. The client could also cache the fact that they have received a callback at their end but this is just another way.